versioned_docs/version-3.10/use-cases/tekton-custom-pipelines.md

```diff
@@ -252,68 +257,85 @@ Now that the Tekton Helm library is created, it is time to clone, modify and the
-    ```
+    ![Create pull request](../assets/use-cases/tekton-custom/tekton-custom15.png "Create pull request")
 
-8. Install the custom chart with the command below. You can also use the `--dry-run` flag to simulate the chart installation and catch possible errors:
+8. Check the GitHub code review for the custom Helm chart pipelines repository in KubeRocketCI. Navigate **Components** -> **Component name** and click to the review pipeline run:
 
-    ```bash
-    helm upgrade --install edp-tekton-custom . -n edp --dry-run
-    ```
+    ![KubeRocketCI pipelines overview](../assets/use-cases/tekton-custom/tekton-custom14.png "KubeRocketCI pipelines overview")
 
-    ```bash
-    helm upgrade --install edp-tekton-custom . -n edp
-    ```
+9. Explore the pipeline status and steps:
 
-9. Check the created pipelines and tasks in the cluster:
+    ![Explore pipeline](../assets/use-cases/tekton-custom/tekton-custom16.png "Explore pipeline")
 
-    ```bash
-    kubectl get tasks -n edp
-    kubectl get pipelines -n edp
-    ```
+10. Go to the GitHub **Pull requests** -> **Add Helm chart testing for go-shell application** -> and click **Merge pull request**:
 
-10. Commit and push the modified Tekton Helm chart to GitHub:
+    ![Merge PR](../assets/use-cases/tekton-custom/tekton-custom17.png "Merge PR")
 
-    ```bash
-    git checkout -b helm
-    git add .
-    git commit -m "Add Helm chart testing for go-shell application"
-    git push -u origin helm
-    ```
+### Deliver Custom Tekton Pipelines to the cluster
 
-11. Navigate Github -> **Pull requests** -> **Compare & pull request** -> **Create pull request**.
+To deploy custom pipelines to the cluster, you can use ArgoCD, which includes a repository with [add-ons](https://github.com/epam/edp-cluster-add-ons/tree/main).
 
-    ![Create pull request](../assets/use-cases/tekton-custom/tekton-custom15.png "Create pull request")
+To deliver custom Tekton pipelines to the cluster, follow these steps:
 
-12. Check the GitHub code review for the custom Helm chart pipelines repository in KubeRocketCI. Navigate **Components** -> **Component name** and click to the review pipeline run:
+1. Clone the forked repository with add-ons and make the following changes. In the `clusters/core/apps/templates/kuberocketci-pipelines.yaml` file, change the `spec.source.repoUrl` field to point to the `tekton-custom-pipelines` repository, for example `ssh://git@github.com:22/<account-name>/custom-tekton-chart.git`. Also, change the `spec.destination.namespace` field to the name of the namespace where the KubeRocketCI platform is deployed.
 
-    ![KubeRocketCI pipelines overview](../assets/use-cases/tekton-custom/tekton-custom14.png "KubeRocketCI pipelines overview")
+    Here is an example of the `kuberocketci-pipelines.yaml` file:
+
+    ```yaml
+    apiVersion: argoproj.io/v1alpha1
+    kind: Application
+    ...
+    spec:
+      source:
+        repoURL: ssh://git@github.com:22/<account-name>/custom-tekton-chart.git
```

Comment:
lets user 'pipelines' instead of 'custom-tekton-chart'

Reply:
Kept the name `custom-tekton-chart` because this aligns with the naming convention used when the library with custom pipelines was created during the `Add Tekton Library` step